README.md

```diff
@@ -23,6 +23,8 @@ Please see the [IRIDA Pipeline Plugin][] documentation for more details.
 | TETyper | <https://github.com/public-health-bioinformatics/irida-plugin-tetyper> | Determine sequence types for specific transposable elements | [![GitHub release](https://img.shields.io/github/release/public-health-bioinformatics/irida-plugin-tetyper.svg)](https://github.com/public-health-bioinformatics/irida-plugin-tetyper/releases/latest) |
 | Snippy Single | <https://github.com/public-health-bioinformatics/irida-plugin-snippy-single> | Variant calling against a reference genome for a single isolate  | [![GitHub release](https://img.shields.io/github/release/public-health-bioinformatics/irida-plugin-snippy-single.svg)](https://github.com/public-health-bioinformatics/irida-plugin-snippy-single/releases/latest) |
 | MiCall-Lite | <https://github.com/public-health-bioinformatics/irida-plugin-micall-lite> | Hepatitis C Virus Genotyping  | [![GitHub release](https://img.shields.io/github/release/public-health-bioinformatics/irida-plugin-micall-lite.svg)](https://github.com/public-health-bioinformatics/irida-plugin-micall-lite/releases/latest) |
+| Plasmid Screen | <https://github.com/public-health-bioinformatics/irida-plugin-micall-lite> | Locate resistance genes on plasmids  | [![GitHub release](https://img.shields.io/github/release/public-health-bioinformatics/irida-plugin-plasmid-screen.svg)](https://github.com/public-health-bioinformatics/irida-plugin-plasmid-screen/releases/latest) |
```

Comment:
Thanks so much @dfornika 👍 .

By the way, it looks like the link <https://github.com/public-health-bioinformatics/irida-plugin-micall-lite> is incorrect in this table?

Reply:
Oops! That's what I get for being lazy and copy-pasting. I'll fix it.

Reply:
Heh. No problem. It happens to everyone.